src/components/inline-inputs/inline-inputs.component.js

```diff
@@ -32,17 +33,33 @@ const InlineInputs = (props) => {
     gutter
   } = props;
 
+  const labelId = useRef(createGuid());
+
   function renderLabel() {
     if (!label) return null;
 
-    return <Label inline htmlFor={ htmlFor }>{ label }</Label>;
+    return (
+      <Label
+        labelId={ labelId }
```

Comment:
you have to access this value by `labelId .current`, `labelId` is an object

Reply:
fixed d7500ff